codigo/trabajoFinalJava/src/trabajofinaljava/TrabajoFinalJava.java

```diff
@@ -0,0 +1,275 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package trabajofinaljava;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author USUARIO
+ */
+public class TrabajoFinalJava {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        // TODO code application logic here
+        Scanner entrada = new Scanner(System.in);
+        String salida;
+        String ciclo;
+        String cadenaFinal="";
+        int contador = 0;
+        int opciones;
+        boolean bandera = true;
+        while (bandera) {
+            System.out.printf("Selecciona 1 y creamos una cuenta de Facebook\n"
+                    + "Selecciona 2 y creamos una cuenta de Twitter\n"
+                    + "Selecciona 3 y creamos  una cuenta de Whatsapp\n"
+                    + "Selecciona 4 y creamos una una cuenta de Telegram\n"
+                    + "Selecciona 5 y creamos una cuenta de Signal\n"
+                    + "Selecciona 6 y creamos una cuenta de Instagram\n"
+                    + "Selecciona 7 y creamos una cuenta de Flickr\n> ");
+            opciones = entrada.nextInt();
+            switch (opciones) {
+                case 1:
+                    cadenaFinal= Facebook();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.